Improve readme (#332)
Erudika/scoold: README.md

	# (optional) require authentication for viewing content
	scoold.is_default_space_public = false
	```
3. Start Scoold with `java -jar -Dconfig.file=./application.conf scoold-*.jar`.
4. Open [localhost:8000/signin/register](http://localhost:8000/signin/register) and
register a new account with same email you put in the configuration.

To login with a social account, you must create a developer app with
[Facebook](https://developers.facebook.com),
[Google](https://console.developers.google.com) or any other identity provider.
This is not required when you log in with LDAP, SAML or email/password.

> misplaced
> Save the obtained API keys in `application.conf`, as shown below.

Author: I was not sure where to put this line, feels diconnected from the content around it.

```suggestion
Save the obtained API keys in `application.conf`, as shown below.
```

> For all identity providers, you must whitelist the Para host with the appropriate authentication endpoint:
> - GitHub: `https://paraio.com/github_auth`  
> - OAuth 2: `https://paraio.com/oauth2_auth`  

> > Learn more about [custom authentication](http://paraio.org/docs/#029-passwordless)  

Author: Do you like the formatting? Is "you must whitelist the Para host with the appropriate authentication endpoint" clear enough? Is it about whitelist authentication? Can you explain?

Reviewer: I think this is clear enough for administrators who configure the OAuth2 apps to work with Para - they need to add the Para host URL to the list of allowed redirects on the configuration page of the OAuth2 provider.

> > See [custom authentication](http://paraio.org/docs/#029-passwordless)  

Author: Not sure about the formatting here. I have added a named like instead of so on. 

Nevertheless, I am not sure if I understand "whitelist the Para host with the appropriate authentication endpoint". I was also confused due to the GitHub Oauth redirect URL.

Can you explain in 1 or 2 sentences?

> > For all identity providers you must whitelist the Para host with the appropriate authentication endpoint, for example: 
> > GitHub redirect URL: `https://paraio.com/github_auth`  
> > OAuth 2: `https://paraio.com/oauth2_auth`  
> > See [custom authentication](http://paraio.org/docs/#029-passwordless)  

Author: I have added a named link instead of "so on" - but not sure about the formatting here. What do you think?

I do not understand "whitelist the Para host with the appropriate authentication endpoint". Can you explain in 1 or 2 sentences? But maybe it's just the word "whitelist" that confuses me, could work out like this: 

```suggestion
> For all identity providers, you must use the correct redirect URL: 
> GitHub: `https://paraio.com/github_auth`  
> OAuth 2: `https://paraio.com/oauth2_auth`  
> Learn more about [custom authentication](http://paraio.org/docs/#029-passwordless)  
```

> > For all identity providers you must whitelist the Para host with the appropriate authentication endpoint, for example: 
> > GitHub redirect URL: `https://paraio.com/github_auth`  
> > OAuth 2: `https://paraio.com/oauth2_auth`  
> > See [custom authentication](http://paraio.org/docs/#029-passwordless)  

Author: ```suggestion
> For all identity providers, you must whitelist the Para host with the appropriate authentication endpoint:
> - GitHub: `https://paraio.com/github_auth`  
> - OAuth 2: `https://paraio.com/oauth2_auth`  
> Learn more about [custom authentication](http://paraio.org/docs/#029-passwordless)  
```

**Scoold requires a persistent and direct connection to a Para server to function properly.**

> <!-- really needed here? The why is missing. Can we highlight them in the example below instead? -->
> <!-- remove or keep --> 
> <!--The most important configuration properties are:
> - `scoold.para_endpoint` - Para server URL
> - `scoold.para_access_key` - Para app application identifier
> - `scoold.para_secret_key` - Para app secret key
> -->

Author: See comments, I think this is not required. para_endpoint, para_access_key, and para_secret_key are in the example below and they are pretty clear. There's some kind of authentication with the host needed. 

Also, the list here explains the para_access_key as application identifier, below access key is used. The word access key should be clear.

I users need this info, I'd go with

```suggestion
Make sure to enter the correct para_access_key, para_secret_key, and para_endpoint for authentication with Para.
```

Author: Otherwise, I'd remove this completely.

```suggestion
```

Reviewer: Since we've added a new auto-configuration feature recently, this part can be removed completely because Scoold can automatically populate these values if that feature is enabled.

</details>

> On startup, Scoold will try to connect to Para 10 times, with a 10 second interval between retries. After that it will

Author: I love my new version here soooo much :)

> On startup, Scoold tries to connect to Para 10 times every 10 seconds (default). If the connection is unsuccessful, the settings will not be persisted. 

Author: ```suggestion
On startup, Scoold tries to connect to Para every 10 seconds, in total 10 times. (default). If the connection is unsuccessful, the settings will not be persisted. 
```

> On startup, Scoold tries to connect to Para every 10 seconds, 10 times in total. (default). If the connection is unsuccessful, the settings will not be persisted. 

Author: I love my new version most here ...

```ini
scoold.connection_retries_max = 10
scoold.connection_retry_interval_sec = 10
```

For an infinite number of attempts to connect to Para, set `connection_retries_max` to `-1`.    

## Docker

> For tagged Docker images for Scoold, go to [erudikaltd/scoold on Docker Hub](https://hub.docker.com/r/erudikaltd/scoold/tags). 

Author: ... and here :)

The `:latest_stable` tag always points to the latest stable release version. The `:latest` tag can be broken/unstable.

> The *easiest way* to create the Scoold stack is to run `docker compose up`.

Author: Will continue here next time if you're happy with my changes :)
